src/main/java/model/data/CombinedDataTable.java

```diff
@@ -123,6 +123,17 @@ public void deleteNotFlagged() {
 		}
 	}
 
+	@Override
+	public DataTable getTable(String name) {
+		if (table.getName().equals(name)) {
```

Comment:
voor de dataTable is de funcie niet echt nodig. Maar de combinedDataTable heeft hem wel nodig. Dus nu kan ik op iedere table getTable doen, zonder eerst te kijken wat voor table het is. Maar misschien is het inderdaad niet netjes dat ik die functie toevoeg en moet ik een functie maken die alle tables uit een combinedDataTable haalt en dan kijkt welke hij terug moet geven